pom.xml

```diff
@@ -416,6 +416,11 @@ under the License.
         <artifactId>javax.annotation-api</artifactId>
         <version>${javaxAnnotationApiVersion}</version>
       </dependency>
+      <dependency>
+        <groupId>org.eclipse.sisu</groupId>
+        <artifactId>org.eclipse.sisu.inject</artifactId>
+        <version>${sisuVersion}</version>
```

Comment:
do we really need both now: `no-asm` and regular?

Reply:
Yes, in modules we use "plain" while the distro uses no_asm + asm dep.

Reply:
Sorry for the noise, missed this is within depMgmt.